### PR TITLE
large-file-upload: initial addition of azure markdown doc

### PR DESCRIPTION
**Pull request by youssifsaeed1**
The added document contains the detailed steps needed to set up MS Azure and send files from the Notecard through Notehub to Azure.

The changes can be properly previewed here:-

https://github.com/blues/note-samples/blob/bf80a63478fff9f0977ab4a0c142f94d3744f4f0/python-large-file-upload/doc/azure.md

### CODE REVIEW

File: python-large-file-upload/doc/azure.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+     <img src="azure_images/azure_generate_sas_key_2.png" style="border: 1px solid lightgray;">
+
+
+6. **Test Azure Container Setup**
+   - Copy the generated SAS URL from the previous step.
+   - Add the name you want your file to be saved as in Azure. For example, if you want to save your file as 'datafile', modify the URL by adding a forward slash "/" after the storage container name, followed by the desired file name. For example, before modifying, the URL looks like this:-
+
+     `
+     https://ysazurestorage.blob.core.windows.net/bluesfilescontainer?sp=racw&st=2024-08-01T11:15:56Z&se=2024-08-01T19:15:56Z&spr=https&sv=2022-11-02&sr=c&sig=Kzo3CL%2BkVm74PTdHM5OoxbjsdHyuc1c1KwDnfkcBjJg%3D
```
Comment (gwolffblues) [inline python-large-file-upload/doc/azure.md:52]: Does this contain any "secrets" or unique IDs?

If so, you may want to replace some of the characters of the unique ID with `xxxxxxxxxxxxxxxxxxxx` or something similar to indicate it's a unique ID, but don't use this specific one.

It's not clear to me, so feel free to ignore this comment if I'm off base.

File: python-large-file-upload/doc/azure.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+     ```
+
+
+     Using the generated link and filename from the example above, the full curl request should look like this:-
+
+     ```bash
+     curl -i -X PUT \
+       -H "x-ms-blob-type: BlockBlob"
+       --upload-file myfile.txt \
+       "https://iotcsayau4ueernijnm.blob.core.windows.net/test-container-20240719/myfile.txt?sp=r&st=2024-07-19T21:20:38Z&se=2024-07-20T05:20:38Z&spr=https&sv=2022-11-02&sr=c&sig=muUfzJZG1K635lTDft0yFyQm4ryQxFU9LsXH4XzrU28%3D"
```
Comment (gwolffblues) [inline python-large-file-upload/doc/azure.md:79]: Contain Unique ID?

**Review by gwolffblues — APPROVED** [review]

File: python-large-file-upload/doc/azure.md (hunk 1 above)
Comment (youssifsaeed1) [reply python-large-file-upload/doc/azure.md:52]: You're right it does, but this specific SAS key was set to expire 24h after it was generated so it is no longer usable.

File: python-large-file-upload/doc/azure.md (hunk 2 above)
Comment (youssifsaeed1) [reply python-large-file-upload/doc/azure.md:79]: Same as above

**Pull request merged**